### DISCUSSION TITLE
Build failed: Gradle build failed with unknown error. See logs for the "Run gradlew" phase for more information.

### DISCUSSION THREAD

**Discussion by TDMTien** [community/community#137177] [Mobile]
### Select Topic Area

Question

### Body

FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring root project 'JobPortalApp'.
> Could not determine the dependencies of null.
   > Could not resolve all task dependencies for configuration ':classpath'.
      > Could not find host.exp.exponent:expo-dev-launcher-gradle-plugin:0.8.0.
        Searched in the following locations:
          - https://dl.google.com/dl/android/maven2/host/exp/exponent/expo-dev-launcher-gradle-plugin/0.8.0/expo-dev-launcher-gradle-plugin-0.8.0.pom
          - https://repo.maven.apache.org/maven2/host/exp/exponent/expo-dev-launcher-gradle-plugin/0.8.0/expo-dev-launcher-gradle-plugin-0.8.0.pom
          - https://maven.google.com/host/exp/exponent/expo-dev-launcher-gradle-plugin/0.8.0/expo-dev-launcher-gradle-plugin-0.8.0.pom
          - https://plugins.gradle.org/m2/host/exp/exponent/expo-dev-launcher-gradle-plugin/0.8.0/expo-dev-launcher-gradle-plugin-0.8.0.pom
        Required by:
            project :
      > Could not find host.exp.exponent:expo-updates-gradle-plugin:0.6.0.
        Searched in the following locations:
          - https://dl.google.com/dl/android/maven2/host/exp/exponent/expo-updates-gradle-plugin/0.6.0/expo-updates-gradle-plugin-0.6.0.pom
          - https://repo.maven.apache.org/maven2/host/exp/exponent/expo-updates-gradle-plugin/0.6.0/expo-updates-gradle-plugin-0.6.0.pom
          - https://maven.google.com/host/exp/exponent/expo-updates-gradle-plugin/0.6.0/expo-updates-gradle-plugin-0.6.0.pom
          - https://plugins.gradle.org/m2/host/exp/exponent/expo-updates-gradle-plugin/0.6.0/expo-updates-gradle-plugin-0.6.0.pom
        Required by:
            project :
* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
* Get more help at https://help.gradle.org/
BUILD FAILED in 55s
Error: Gradle build failed with unknown error. See logs for the "Run gradlew" phase for more information.

**Comment by TranDangMyTien** [comment]
{
  "name": "jobportalapp",
  "version": "1.0.0",
  "main": "index.js",
  "scripts": {
    "start": "expo start --dev-client",
    "android": "npx expo run:android",
    "ios": "npx expo run:ios",
    "web": "expo start --web",
    "buildAAB": "eas build -p android --profile production",
    "buildAPK": "eas build -p android --profile preview"
  },
  "dependencies": {
    "@expo/vector-icons": "^14.0.2",
    "@react-native-async-storage/async-storage": "^1.23.1",
    "@react-native-community/datetimepicker": "^8.0.1",
    "@react-native-community/masked-view": "^0.1.11",
    "@react-native-firebase/app": "^20.4.0",
    "@react-native-firebase/auth": "^20.4.0",
    "@react-native-google-signin/google-signin": "^12.2.1",
    "@react-native-picker/picker": "^2.7.5",
    "@react-native/assets": "^1.0.0",
    "@react-navigation/bottom-tabs": "^6.5.20",
    "@react-navigation/drawer": "^6.7.0",
    "@react-navigation/native": "^6.1.17",
    "@react-navigation/stack": "^6.3.29",
    "axios": "^1.7.2",
    "electron": "^32.0.1",
    "env-cli": "^1.0.5",
    "expo": "^51.0.31",
    "expo-constants": "~16.0.1",
    "expo-dev-client": "~4.0.25",
    "expo-dev-launcher": "^4.0.27",
    "expo-font": "^12.0.9",
    "expo-image-picker": "^15.0.7",
    "expo-linear-gradient": "^13.0.2",
    "expo-linking": "~6.3.1",
    "expo-modules-autolinking": "^1.11.2",
    "expo-modules-core": "^1.12.23",
    "expo-router": "^3.5.23",
    "expo-splash-screen": "^0.27.5",
    "expo-status-bar": "~1.12.1",
    "expo-updates": "^0.25.24",
    "iconsax-react-native": "^0.0.8",
    "ip": "^2.0.1",
    "lottie-react-native": "^6.7.0",
    "metro-config": "^0.80.10",
    "metro-react-native-babel-preset": "^0.77.0",
    "moment": "^2.30.1",
    "node": "^22.5.0",
    "react": "^18.2.0",
    "react-native": "^0.75.2",
    "react-native-asset": "^2.1.1",
    "react-native-config": "^1.5.1",
    "react-native-eject": "^0.2.0",
    "react-native-elements": "^3.4.3",
    "react-native-fs": "^2.20.0",
    "react-native-gesture-handler": "^2.16.1",
    "react-native-image-picker": "^7.1.2",
    "react-native-image-zoom-viewer": "^3.0.1",
    "react-native-loading-spinner-overlay": "^3.0.1",
    "react-native-modal": "^13.0.1",
    "react-native-paper": "^5.12.3",
    "react-native-reanimated": "^3.10.1",
    "react-native-render-html": "^6.3.4",
    "react-native-safe-area-context": "^4.10.5",
    "react-native-safe-area-view": "^1.1.1",
    "react-native-screens": "^3.20.0",
    "react-native-svg": "^15.2.0",
    "react-native-swiper": "^1.6.0",
    "react-native-vector-icons": "^10.1.0",
    "react-navigation": "^5.0.0",
    "rn-nodeify": "^10.3.0",
    "watchman": "^1.0.0"
  },
  "devDependencies": {
    "@babel/core": "^7.24.7",
    "@expo/ngrok": "^4.1.3",
    "@types/react-native-vector-icons": "^6.4.18",
    "react-devtools": "^4.27.2",
    "react-native-dotenv": "^3.4.11"
  },
  "private": true
}

**Comment by ebndev** [comment]
Please see [this comment](https://github.com/orgs/community/discussions/137178#discussioncomment-10501195) for reference to why this discussion was closed and locked. Thanks!
